app/views/users/show.html.erb

```diff
@@ -1,2 +1,31 @@
-<h1> Here is an author with its corresponding bio and his/her posts
-</h1>
+<div class="user-posts-wrapper">
+  <h1 class="header-text"> Here is an author with her/his corresponding bio and his/her posts </h1>
```

Comment:
- [ ] Please remove this line to match the given wireframe.

Reply:
lines removed.